components/nav/MobileNav.tsx

```diff
@@ -32,6 +32,10 @@ const MobileNav = () => {
         }
     }, [stopScroll])
 
+    const route = (value: string) => {
```

Comment:
If this function is not being used anywhere else, which it looks like it isn't, there's no need to make it 'reusable'.

```
const routeToHome = () => {
 setShow(false)
 router.push('/')
}